src/components/smart/Bluetooth.js

```diff
@@ -131,13 +180,24 @@ export default class Bluetooth extends Component {
      */
     async readBattery(id){
         /// code expected
-        throw new Error("Bluetooth: read battery level not implemented");
+        //throw new Error("Bluetooth: read battery level not implemented");
+
+        await BleManager.read(id, "0x180D", "2A39")
+            .then((readData) => {
+                // Success code
+                console.log('Read : ' + readData);
+                const buffer = Buffer.Buffer.from(readData);
+                const sensorData = buffer.readUInt8(1, true);
+            })
```

Comment:
Didn't add `catch` and didn't return battery level. No point in using `await` with `.then`.
0/1 and 0/1